README.md

```diff
@@ -118,6 +118,34 @@ ngrok http 8443 -host-header="localhost:8443"
 
 The frontend app is composed of a number of components that are documented in our [Storybook](https://wordpress.github.io/openverse-frontend).
 
+##### How to create a component
+
+To create a component you can run:
+
+```
+pnpm run create-component [component name in camel case] [component name in kebab-case]
```

Comment:
I realized that we need `PascalCase`, and not `camelCase` for the first instance of the component name.